Blockzilla/AutocompleteCustomUrlViewController.swift

```diff
@@ -109,9 +109,9 @@ extension AutocompleteCustomUrlViewController: UITableViewDataSource {
 
             let backgroundColorView = UIView()
             backgroundColorView.backgroundColor = UIConstants.colors.cellSelected
-
             cell.selectedBackgroundView = backgroundColorView
             addDomainCell = cell
+            if self.tableView.isEditing { cell.isHidden = true }
```

Comment:
@deepal15 Would you mind simplifying this to `cell.isHidden = self.tableView.isEditing` ? Let us know if you want to finish this PR.